README.md

```diff
@@ -97,7 +97,7 @@ configurations.
 ### Misc Targets
 * `make clean`: Delete binaries built from source
 =======
-* `make run`: Runs the broker with the default profile, configured via `/etc/broker-config.yaml`
+* `make run`: Runs the broker with the default profile, configured via `/etc/config.yaml`
```

Comment:
I'd change this to:

 `make run`: Runs the broker with a generated configuration file